src/containers/CreateContact/index.js

```diff
@@ -0,0 +1,158 @@
+      }
+      setTimeout(() => {
+        this.setState({ submitted: false });
+      }, 3000);
+
+    });
+  }
+
+
+
+  render() {
+
+    return (
+      <div >
+        <div style={{ display: "flex", justifyContent: "center" }}>
+          {this.state.success && this.state.submitted ?
+            <Alert isOpen={this.state.visible} style={{ width: "48rem" }} variant='success'> Successful contact creation!</Alert>
+            : !this.state.success && this.state.submitted ? <Alert style={{ width: "48rem" }} variant='danger'> Error!</Alert> : ''}
+        </div>
+        <div style={{ display: "flex", justifyContent: "center" }}>
+
+
+          <Card style={{ width: "48rem" }}>
+            <Card.Header as="h5">Create Contact</Card.Header>
+            <Card.Body>
+              <div className="Login">
+                <Form >
+                  <FormGroup bsSize="large">
+                    <FormControl
+                      onChange={(e) => this.setState({ contactName: e.target.value })}
+                      placeholder="Name"
+                    />
+                  </FormGroup>
+
+                  <br></br>
+                  <FormGroup bsSize="large">
+                    <FormControl
+                      onChange={(e) => this.setState({ email: e.target.value })}
+                      placeholder="Email"
+                    />
+                  </FormGroup>
+                  <br></br>
+                  <FormGroup bsSize="large">
+                    <FormControl
+                      placeholder="Phone"
+                      onChange={(e) => this.setState({ phone: e.target.value })}
+                    />
+                  </FormGroup>
+                  <br></br>
+                  <FormGroup bsSize="large">
+                    <FormControl
+                      placeholder="Address"
+                      onChange={(e) => this.setState({ address: e.target.value })}
+                    />
+                  </FormGroup>
+                  <br></br>
+                  <Row>
+                    <Col>
+                      <Form.Control
+                        placeholder="City"
+                        onChange={(e) => this.setState({ city: e.target.value })}
+                      />
+                    </Col>
+                    <Col>
+                      <Form.Control
+                        placeholder="State"
+                        onChange={(e) => this.setState({ state: e.target.value })}
+                      />
+                    </Col>
+                    <Col>
+                      <Form.Control
+                        placeholder="Zip"
+                        onChange={(e) => this.setState({ zipCode: e.target.value })}
+                      />
+                    </Col>
+                  </Row>
+                  <br></br>
+                  <FormGroup bsSize="large">
+                    <FormControl
+                      placeholder="Institution"
```

Comment:
Might be helpful to add inst details in the form too...name, address etc. Please take a look at the createDonor page in the current master branch as an example. I've done some backend changes to support this. It might not be helpful if the user has to create an institution separately and then create a contact. I actually think that the creation of contacts and inst will mostly happen in the create donor page anyway....But they could still add potential contacts in the contacts page.